src/Headers/ContentSecurityPolicy/Domain.php

```diff
@@ -0,0 +1,35 @@
+<?php declare(strict_types = 1);
+
+namespace Pd\SecurityHeaders\Headers\ContentSecurityPolicy;
+
+final class Domain implements IValue
+{
+
+	/**
+	 * @var string
+	 */
+	private $domain;
+
+
+	public function __construct(string $domain)
+	{
+		$domainForCheck = $domain;
+
+		if (\strpos($domain, '*.') === 0) {
+			$domainForCheck = \substr($domainForCheck, 2);
+		}
+
+		if ( ! \Nette\Utils\Validators::isUrl('http://' . $domainForCheck)) {
```

Comment:
Ta ta validace domény bude asi trochu složitější. Hvězdička totiž může být použita pro nejlevější část hostname, pro port a pro protokol. Takže např. `*://*.foo.bar.com:*` je taky v CSP validní. Viz třeba https://scotthelme.co.uk/content-security-policy-an-introduction/, tam se o tom píše. Možná bych tu validaci nakonec odstranil, zapoměl jsem, že tam můžou být ty hvězdičky. Nebo bych to udělal pořádně, ale teď mě nenapadá žádné jednoduché řešení.
Ty tam potom ještě přidáváš `http://`, ale to už tam může být uvedené.